### ISSUE TITLE
Server Connection Issue?

### ISSUE THREAD

**Issue by ahdeck** [raspberrypi/rpi-imager#312]
Am I the only one having this issue? It's been going on for a couple of days. I'm not having any internet issues on my end as far as I know, and my download speed it currently 175mbps. Maybe it's just a server problem for them, to be resolved after the weekend?

![Untitled](https://user-images.githubusercontent.com/95054994/143504039-2a4b161a-f399-4e72-9f31-1c2725af56c3.png)

**Comment by maxnet** [comment]
Seems it is trying to connect over IPv6 in your case.

https://test-ipv6.com/ shows your IPv6 connectivity is good?

**Comment by ahdeck** [comment]
Thanks for your response! And it looks like you are onto something. When I run the test, everything passes except for the last category:
"Your DNS server (possibly run by your ISP) appears to have no access to the IPv6 Internet, or is not configured to use it. This may in the future restrict your ability to reach IPv6-only sites."

So, I guess the next question is...how do I get the imager to use a compatible protocol? I am just a noob with a pi.

**Comment by maxnet** [comment]
Try if it work better if you disable IPv6 globally: https://networking.grok.lsu.edu/article.aspx?articleid=17573

**Comment by ahdeck** [comment]
Hope it was a good holiday.

That was easy to do. Unfortunately, no dice.

![Untitled](https://user-images.githubusercontent.com/95054994/143979101-7def928b-96cc-493c-8f17-cd06c93e2179.png)

I'm going to be upgrading my internet equipment this week. Let me try again after that. That still seems strange though, it's not like I'm on dial-up or satellite from a remote part of Alaska. So, if you think of anything else in the meantime, please let me know.

Thank you for your help as always.

**Comment by maxnet** [comment]
>Hope it was a good holiday.

Am afraid not so in Europe.
(Most get plenty of vacation days to make up for our lack of national holidays though)

>That was easy to do. Unfortunately, no dice.

Are you sure you only unticked IPv6 (and not accidently IPv4 as well)?

**Comment by ahdeck** [comment]
Hadn't disabled IPv4, but with a new wireless adapter (USB AC600 to PCE AC1900) the problem seems to be solved. ¯\_(ツ)_/¯

Thanks for all your help!

**Comment by tdewey-rpi** [comment]
Closing as won't fix - problem appears to have been related to a HW configuration.
